wordpress-mobile/WordPress-iOS
# Issue/16957 - Display scan error states

Fixes [#16957](https://github.com/wordpress-mobile/WordPress-iOS/issues/16957)

### Description

This PR displays appropriate errors using `NoResultsViewController`.

### Result

- Multisite not supported error

Before | After
--- | ---
<img width="509" alt="Screen Shot 2021-08-10 at 09 04 10" src="https://user-images.githubusercontent.com/12729242/128823518-a64b0e59-5d82-4a8d-a9c7-5a53f7efb560.png"> | <img width="509" alt="Screen Shot 2021-08-09 at 16 56 39" src="https://user-images.githubusercontent.com/12729242/128823576-8948dfa4-b6ec-4868-ac38-c0d5e09301da.png">

### How to test

- Multisite not supported error

1. Go to the Jurassic Ninja special ops page
2. Select the **Jetpack Complete** plan
3. Launch a multisite on subdomains
4. Go to newly created site dashboard and click on **Set up Jetpack**
5. Inside Jetpack / Wordpress app select newly created site
6. Tap on **My Site** > **Scan**
6. Verify new design

### Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

### PR submission checklist:
